1_space_odyssey/index.html

```diff
@@ -9,8 +9,8 @@
     <title>2020: A Space Odyssey</title>
 </head>
 <body class="fade-in">
-    
-    <script src="./js/mannuel-9000/app.js"></script>
+<script src="js/mannuel-9000/ben-9000/app.js"></script>    
+  
     <!-- include your javascript file here -->
```

Comment:
please put my JS include back and add yours after mine ;-)

Reply:
Did just fix it, help from you:)